MUONdigi/dataFormatExtension/muonSystemHit.yaml

```diff
@@ -0,0 +1,21 @@
+---
+schema_version: 1
+options:
+  # should getters / setters be prefixed with get / set?
+  getSyntax: True
+  # should POD members be exposed with getters/setters in classes that have them as members?
+  exposePODMembers: False
+  includeSubfolder: True
+
+datatypes:
+
+  extension::MCRecoMuonSystemDigiAssociation:
```

Comment:
Why not using `MCRecoTrackerAssociation` https://github.com/key4hep/EDM4hep/blob/main/edm4hep.yaml#L634 ?

Reply:
Can it be used for linking between digi and sim? I fount it relates between reco and sim.. 
On another hand I'd like to see if there is an association also to store the validation data (e.g: simDigipPositionDifference)